project/backend/src/main/java/com/google/sps/data/RestaurantScorer.java

```diff
@@ -37,11 +53,29 @@ public static double score(Restaurant restaurant, JSONObject preferences) throws
     return percentMatch;
   }
 
+  /** Calculates the distance in meters between two sets of lat long coordinates. */
```

Comment:
nit: add "....... lat long coordinates using the Haversine formula." 

(useful to add it to comments for future readers to read more about it :) )